js/wp-seo-metabox.js

```diff
@@ -223,10 +223,13 @@ function yst_updateTitle(force) {
 		var placeholder_title = divHtml.html(title).text();
 		titleElm.attr('placeholder', placeholder_title);
 
+		// Run possibly set filters
+		title = wpseo_apply_filter( title, 'title');
```

Comment:
@andizer For the description you have created `sanitize_desc`. Why wouldn't you do the same for the title?